.github/workflows/ci-schema.yml

```
@@ -0,0 +1,11 @@
+name: Check Schema
+
+on:
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+
+jobs:
+  check:
+    uses: Lightning-AI/devtools/.github/workflows/check-schema.yml@main  # FIXME: Use v0.1.0
```

TODO: Apply this suggestion once https://github.com/Lightning-AI/devtools/pull/15 is merged.
```suggestion
    uses: Lightning-AI/devtools/.github/workflows/check-schema.yml@v0.1.0
```